src/main/java/net/minecraftforge/registries/ObjectHolderRegistry.java

```diff
@@ -171,7 +171,23 @@ public static void applyObjectHolders()
 
     public static void applyObjectHolders(Predicate<ResourceLocation> filter)
     {
-        objectHolders.forEach(e -> e.accept(filter));
+        RuntimeException aggregate = new RuntimeException("Failed to apply some object holders, see suppressed exceptions for details");
+        objectHolders.forEach(objectHolder -> {
+            try
+            {
+                objectHolder.accept(filter);
+            }
+            catch (Exception e)
+            {
+                aggregate.addSuppressed(e);
+            }
+        });
+
+        if (aggregate.getSuppressed().length > 0) {
```

Comment:
Formatting: braces on newlines.
```suggestion
        if (aggregate.getSuppressed().length > 0)
        {
```